services/logging_service.py

```
@@ -0,0 +1,292 @@
+                    f"Failed to send log to remote collector: "
+                    f"HTTP {response.status_code}"
+                )
+                return False
+
+        except requests.exceptions.Timeout:
+            self.logger.warning("Timeout sending log to remote collector")
+            return False
+        except requests.exceptions.RequestException as e:
+            self.logger.warning(f"Error sending log to remote collector: {e}")
+            return False
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)